ui/components/ui/box/box.js

```diff
@@ -264,6 +264,7 @@ Box.propTypes = {
   display: PropTypes.oneOf(Object.values(DISPLAY)),
   width: PropTypes.oneOf(Object.values(BLOCK_SIZES)),
   height: PropTypes.oneOf(Object.values(BLOCK_SIZES)),
-  backgroundColor: PropTypes.oneOf(Object.values(ValidBackgroundColors)),
+  // backgroundColor: PropTypes.oneOf(Object.values(ValidBackgroundColors)),
```

Comment:
🤔

Reply:
Yeah I have no clue why this didn't cause e2e to fail before, but upon running the e2e test that was failing on this branch, which centers around editing gas fees, I am now getting:

```
Warning: Failed prop type: Invalid prop `backgroundColor` of value `white` supplied to `Box`, expected one of ["background-default","background-alternative","overlay-default","primary-default","primary-alternative","primary-muted","primary-disabled","secondary-default","secondary-alternative","secondary-muted","secondary-disabled","error-default","error-alternative","error-muted","error-disabled","warning-default","warning-alternative","warning-muted","warning-disabled","success-default","success-alternative","success-muted","success-disabled","info-default","info-alternative","info-muted","info-disabled","mainnet","ropsten","kovan","rinkeby","goerli","transparent","localhost"]
```

This appears to be happening because the default `backgroundColor` is `"white"` when it should be one of the values specified above. I think we should probably change the default to `"background-default"` but I'm afraid that'll break things.

Commenting this line out is definitely not the best solution but I can look at this in a future PR.

Reply:
Yeah I think we swallow the warning for now and figure out whether/when we change the default to `background-default`. It shouldn't be causing a failure? I don't think we should do this...